content/docs/use-cases/fast-data-storage-layer.md

```diff
@@ -0,0 +1,143 @@
+
+DVC's built-in data <abbr>caching</abbr> enables you to implement a simple and
+efficient storage layer globally — for all your team's projects. This approach
+can help in situations such as:
+
+- You want to speed up data transfers from a massive object store which is
+  currently on the cloud, or to move a growing dataset out of your machine
+  without slowing things down. How can you use something in the middle (like a
+  NAS drive)?
+
+- Upgrading your entire storage platform is expensive. Can you pay only for fast
+  access to frequently-used data?
+
+- You want to de-duplicate files automatically when multiple people are working
+  on the same data (for example, on a
+  [shared development server](#example-shared-development-server)).
+
+- Your team shares access to a GPU server for machine learning
+  [experiments](/doc/user-guide/experiment-management). How can you switch the
+  data inputs quickly, without re-downloading every time?
+
+<abbr>DVC projects</abbr> transform raw data, intermediate artifacts, and final
+results (e.g. machine learning models) into unique storage objects. This
+optimizes storage by avoiding file duplication. And to minimize transfers,
+<abbr>cached</abbr> files are [linked] back to your <abbr>workspace</abbr>.
+
+> 📖 See [Dataset Optimization](/doc/user-guide/large-dataset-optimization) for
+> more details.
+
+[linked]:
+  /doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache
+
+**You can unify all your data across projects by setting up a [shared cache] as
+primary storage.** This compounds the deduplication and performance benefits.
+Having separated data storage from other aspects of your projects, you can treat
+it as an independent infrastructure layer which can use Ceph,
+[EFS](https://aws.amazon.com/efs/), etc. depending on the access speed and
+frequency required.
+
+![](/img/shared-server.png) _Data storage shared by DVC projects_
+
+<!-- Simplify/ make more general? Not local, symlinks only -->
+
+To balance speed and cost, you can also use a `dvc remote` (or several) as
+secondary storage (e.g. an object store like S3,
+[GCS](https://cloud.google.com/storage/), or Azure Blob Storage). Data only
+needed in older [versions](/doc/use-cases/versioning-data-and-model-files) of
+your project can be kept there. Remote storage is also a great way to back up
+and [share datasets and ML models](/doc/use-cases/sharing-data-and-model-files).
+
+> 🔒 Another benefit of having a clear organization of your near and remote
+> storage locations is to manage user access, as well as centralizing data
+> security policies.
+
+Using DVC to implement a layered data storage system also gives you flexibility
+to redesign the solution in the future without having to change directory
+structures or code.
+
+<details>
+
+## Click to explore a range of advanced implementations
+
+- On one extreme, you can [prepare your data requirements] in a lightweight
```

Comment:
This chunk is a) a conclusion and b) a last bit about going from Layer to Layers and advanced possibilities for this pattern. May needs work still (or removing)

Reply:
p.s. covers the simple cache project scenario mentioned in https://github.com/iterative/dvc.org/pull/2587#discussion_r671570947.